docs/resources/lakehouse_monitor.md

```diff
@@ -0,0 +1,135 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_lakehouse_monitor Resource
+
+This resource allows you to manage [Lakehouse Monitors](https://docs.databricks.com/en/lakehouse-monitoring/index.html) in Databricks. 
+
+A `databricks_lakehouse_monitor` is attached to a [databricks_sql_table](sql_table.md) and can be of type timeseries, snapshot or inference. 
```

Comment:
It can be attached to any UC table right, not just this specific SQL table which is a workaround to initialize tables via Terraform? Maybe we should call out that it can be used with the table data source as well